rails-commenting.rb

```diff
@@ -7,24 +7,29 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+#  BlogPost will inherit all methods from ApplicationController
 class BlogPostsController < ApplicationController
   def index
     # ---2)
+    # will call on the methods (posts) to contain in the blogpost
```

Comment:
This will be storing all of the blog posts inside an instance variable called posts.